src/cdk-experimental/listbox/listbox.ts

```diff
@@ -105,6 +109,29 @@ export class CdkListbox<V> {
     items: this.items,
     textDirection: this.textDirection,
   });
+
+  /** Whether the listbox has received focus yet. */
+  private _touched = signal(false);
+
+  /** Whether the options in the listbox have been initialized. */
+  private _isViewInitialized = signal(false);
+
+  constructor() {
+    effect(() => {
+      if (this._isViewInitialized() && !this._touched()) {
+        const index = this.items().findIndex(i => this.value().includes(i.value()));
```

Comment:
Feels weird to have these signal-reads knowing that it causes this effect to be run a lot. I guess it doesn't matter because of the initial conditional check, so its a fast function. But I wish there was a way for Angular to know this should really only be run based on the two conditional signals

Reply:
I mean the `this.items()`, `this.value()`, `i.value()` - these will all trigger the effect right?

Reply:
Oh, yes but those are needed to trigger the effect in order to figure out the correct start index. The effect should rerun any time those signals change until the listbox is touched (focused for the first time)

Reply:
> But I wish there was a way for Angular to know this should really only be run based on the two conditional signals

I think you can wrap the rest of the reads in `untracked(() => ...)` to indicate that (though it sounds like maybe you do need it to react to all of them here, just wanted to give an fyi that the feature exists)